### ISSUE TITLE
Missing all pac-man games

### ISSUE THREAD

**Issue by Nomelas** [tranxuanthang/villain3ds#27]
Pac-Man and the Ghostly Adventures - 00040000000f1200
Pac-Man and the Ghostly Adventures 2 - 0004000000135f00

actually, none of the pac-man titles show up at all including the VC ones

Also, this is not 'correct':
Kirby's Blowout Blast - 004000000196F00

It's an all region title, but marked as JPN only, and shows as: カービィのすいこみ大作戦

**Comment by tranxuanthang** [comment]
I've just updated the database so check it again.
![Imgur](https://i.imgur.com/1czALI1.png)

**Comment by Nomelas** [comment]
how do i refresh the db?

**Comment by Nomelas** [comment]
oh, just searching "PAC-MAN" doesn't return results, need to add more words it seems.

Thanks for that!

also, what about the kirby game?

**Comment by tranxuanthang** [comment]
Yeah I don't know why searching PAC-MAN doesn't return any results, maybe a new bug.
According to 3dsdb, 004000000196F00 is Kirby no Suikomi Daisakusen, JPN region. Are you sure it's wrong?

**Comment by Nomelas** [comment]
yes, that's also wrong, the 4 letter game id is JHUA - A = ALL

**Comment by Nomelas** [comment]
004000000196F00 is actually a shared titleID between regions
